Makefile

```diff
@@ -4,14 +4,14 @@ HELM_PLUGIN_DIR ?= $(HELM_HOME)/plugins/helm-2to3
 VERSION := $(shell sed -n -e 's/version:[ "]*\([^"]*\).*/\1/p' plugin.yaml)
 DIST := $(CURDIR)/_dist
 LDFLAGS := "-X main.version=${VERSION}"
-MOD_PROXY_URL ?= https://gocenter.io
+MOD_PROXY_URL ?= https://goproxy.io
 
 PHONY: bootstrap
 bootstrap:
 	export GO111MODULE=on && \
 	export GOPROXY=$(MOD_PROXY_URL) && \
-	go mod download
-
+	go mod download && \
+	export GOOS=darwin && export GOARCH=amd64
```

Comment:
not really necessary changes

Reply:
yes indeed, it should be already removed by 2nd commit